llvm/lib/Passes/PassRegistry.def

```diff
@@ -242,17 +242,17 @@ FUNCTION_ANALYSIS("no-op-function", NoOpFunctionAnalysis())
 FUNCTION_ANALYSIS("opt-remark-emit", OptimizationRemarkEmitterAnalysis())
 FUNCTION_ANALYSIS("pass-instrumentation", PassInstrumentationAnalysis(PIC))
 FUNCTION_ANALYSIS("phi-values", PhiValuesAnalysis())
-FUNCTION_ANALYSIS("postdomtree", PostDominatorTreeAnalysis())
+FUNCTION_ANALYSIS("post-domtree", PostDominatorTreeAnalysis())
```

Comment:
This hyphenation doesn't make sense to me. It should be either `postdomtree` or `post-dom-tree`, but not something in the middle.